Skip django tests as it is not in void-packages
--- a/tests/debugpy/test_django.py
+++ b/tests/debugpy/test_django.py
@@ -45,6 +45,7 @@
 
 
 @pytest.mark.parametrize("bp_target", ["code", "template"])
+@pytest.mark.skip(reason="django is not packaged by void")
 def test_django_breakpoint_no_multiproc(start_django, bp_target):
     bp_file, bp_line, bp_name = {
         "code": (paths.app_py, lines.app_py["bphome"], "home"),
@@ -90,6 +91,7 @@
             assert bp_var_content in home_request.response_text()
 
 
+@pytest.mark.skip(reason="django is not packaged by void")
 def test_django_template_exception_no_multiproc(start_django):
     with debug.Session() as session:
         with start_django(session):
@@ -136,6 +138,7 @@
 
 
 @pytest.mark.parametrize("exc_type", ["handled", "unhandled"])
+@pytest.mark.skip(reason="django is not packaged by void")
 def test_django_exception_no_multiproc(start_django, exc_type):
     exc_line = lines.app_py["exc_" + exc_type]
 
@@ -186,6 +189,7 @@
             session.request_continue()
 
 
+@pytest.mark.skip(reason="django is not packaged by void")
 def test_django_breakpoint_multiproc(start_django):
     bp_line = lines.app_py["bphome"]
     bp_var_content = "Django-Django-Test"
